### DISCUSSION TITLE
Edit button for Artist and Album Art

### DISCUSSION THREAD

**Discussion by theuniversal2017** [fast4x/RiMusic#1556] [Ideas]
I know this you can do it on the Artist profile, but the changes doesn't save. And, I think this can be added for songs than doesn't in the app (audio)
 
Please, I like add this option for the songs when are added in Favourites item and, can be saved. I hope add on the next actualization. 

Thanks.

**Comment by fast4x** [comment]
Hi, sure there are many tickets about this, I'll fix soon.
